.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
No, I don't insist, but IMO projects shouldn't be concerned with the used IDE or editor.